File: solutions/clojure/resistor-color-duo/1/src/resistor_color_duo.clj
(ns resistor-color-duo)

(def color-map {"black" 0, "brown" 1, "red" 2, "orange" 3, 
                "yellow" 4, "green" 5, "blue" 6, "violet" 7,
                "grey" 8, "white" 9})

(defn resistor-value
  "Returns the resistor value based on the given colors"
  [colors]
   (+ (* 10 (get color-map (nth colors 0))) 
              (get color-map (nth colors 1))))
  

File: solutions/julia/resistor-color-duo/1/resistor-color-duo.jl
color_dict = 
    Dict("black"=>0, "brown"=>1, "red"=>2, "orange"=>3,
        "yellow"=>4, "green"=>5, "blue"=>6, "violet"=>7,
        "grey"=>8, "white"=>9)

function colorcode(colors)
    n1= color_dict[colors[1]]
    n2 = color_dict[colors[2]]
    ohms = 10*n1 + n2
end
